.github/workflows/release.yml

```diff
@@ -11,13 +11,46 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
-        django: ['3.2', '4.0', '4.1']
```

Comment:
I realized this duplicated actions file wasn't updated in the past 😕. Maybe there’s a way to reduce the duplication more.

Reply:
I looked for ways to move the matrix into the task but I couldn't find any. It's probably OK to do a significantly reduced set here, anyway